src/App.vue

```diff
@@ -160,4 +160,26 @@ export default {
 #app {
 	height: 100%;
 }
+
+// Kludges to ensure Kiva Classic styles are correct when using the .kv-tailwind class
+// Remove once Kiva Classic is the default
+.kv-tailwind {
+	p,
```

Comment:
The existing reset sets line-heights on these elements to 1.75.

Tailwind's reset doesn't specify a line-height on these elements, so they were using the old system.